refgenconf/refgenconf.py

```diff
@@ -1765,7 +1778,9 @@ def list_seek_keys_values(self, genomes=None, assets=None):
                 for tag_name in get_asset_tags(asset_mapping):
                     tag_mapping = asset_mapping[CFG_ASSET_TAGS_KEY][tag_name]
                     ret[genome_name][asset_name][tag_name] = {}
-                    for seek_key_name in get_tag_seek_keys(tag_mapping):
+                    if not (tag_seek_keys := get_tag_seek_keys(tag_mapping)):
```

Comment:
While it is possible to use the walrus operator in this case, I would advise against it. Because the variable `tag_seek_keys` is used outside the `if` statement, it is kind to the reader to also define it outside the if statement. The if statement also becomes much easier to read.

```suggestion
                    tag_seek_keys = get_tag_seek_keys(tag_mapping)
                    if not tag_seek_keys:
```